src/timeMachine/components/visualization/ViewTypeDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {FC} from 'react'
+import {TYPE_DEFINITIONS} from 'src/shared/visualization'
+
+// Components
+import {Dropdown, DropdownMenuTheme} from '@influxdata/clockface'
+import {ComponentStatus} from 'src/clockface'
+
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
```

Comment:
why is this file here instead of using the `src/shared/visualization` version?

Reply:
Good catch, thanks!